wiki/Katu/en.md

```diff
@@ -9,6 +8,56 @@ tags:
 
 *See also: [Geki](/wiki/Geki)*
 
-**Katu (喝)**, *Katsu*, or *Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset without achieving the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note. This idea originates from the Nintendo DS game Elite Beat Agents which is one of the games the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+*Not to be confused with osu!taiko's Katu*
 
-<!-- TODO: Add links -->
+**Katu (喝)**, *Katsu*, or *Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset without achieving the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note.
+However, this can't be achieved by having a 50 or lower in the combo itself.
+
+There are two kind of Katu, which is based on the combo finishing judgment. A 300-ended Katu rewards more health bonus than 100-ended Katu, but always worse than a Geki.
+
+This idea originates from the Nintendo DS game Elite Beat Agents which is one of the games the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+
+## Screenshots
+
+In here provided the screenshots of obtaining a Katu/Katsu/Beat from Elite Beat Agents, Ouendan 2, and osu! itself.
+
+![](img/eba-without-100k.jpg) ![](img/oto-countdwn-300k.jpg)
+
+![](img/osu-cutepnth-100k.jpg)
+
+![](img/osu-stardisc-300k.jpg)
+
+## Example gameplay links
+
+- [Elite Beat Agents: Without a Fight/Jumpin' Jack Flash - YouTube](https://www.youtube.com/watch?v=wPOCmzY_fqs)
+- [Osu! Tatakae! Ouendan 2: Countdown - YouTube](https://www.youtube.com/watch?v=6us1tY8jOSI)
+
+## Other game modes
+
+### osu!taiko
+
+Katu is the judgment given when you hit big notes at same time. osu!taiko have more leniency on handling both health bonus and score bonus for hitting both keys within short timespan.
```

Comment:
I don't get the second sentence, mind explaining what it is about or rephrasing it?
```suggestion
Katu is the judgement given when a player hits big notes perfectly by pressing two keys of the same colour simultaneously. osu!taiko has more leniency on handling both health bonus and score bonus for hitting both keys within short timespan.
```

Reply:
i forgot to think the word "simulatenously" for no reason... thanks